packages/common/src/lib/common/tools.ts

```diff
@@ -1250,16 +1291,20 @@ function getRepositoryFile(urlOrPath, additionalInfo, callback) {
  *
  * @alias getRepositoryFileAsync
  * @memberof tools
- * @param {string} url URL starting with http:// or https:// or local file link
- * @param {string} hash actual hash
- * @param {boolean} force Force repository update despite on hash
- * @param {object} _actualRepo Actual repository
+ * @param url URL starting with http:// or https:// or local file link
+ * @param hash actual hash
+ * @param force Force repository update despite on hash
+ * @param _actualRepo Actual repository
  *
  */
-async function getRepositoryFileAsync(url, hash, force, _actualRepo) {
+async function getRepositoryFileAsync(
+    url: string,
+    hash: string,
+    force: boolean,
+    _actualRepo: Record<string, any>
+): Promise<Record<string, any>> {
     let _hash;
     if (_actualRepo && !force && hash && (url.startsWith('http://') || url.startsWith('https://'))) {
-        axios = axios || require('axios');
         _hash = await axios({ url: url.replace(/\.json$/, '-hash.json'), timeout: 10000 });
```

Comment:
We're using `axios` here, but `request` elsewhere. We should get rid of `request`.

Reply:
yes, but not necessarily in this PR.